agents-api/agents_api/activities/execute_integration.py

```diff
@@ -9,7 +9,7 @@
 from ..common.exceptions.tools import IntegrationExecutionException
 from ..common.protocol.tasks import ExecutionInput, StepContext
 from ..env import testing
-from ..queries.tools import get_tool_args_from_metadata
+from ..queries import tools
```

Comment:
Is there a reason behind importing queries in this way? It's not consistent with the rest of the files, and a bit counter-intuitive

Reply:
This goes for other queries in this PR too